linkis-dist/docker/ldh.Dockerfile

```diff
@@ -0,0 +1,78 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+######################################################################
+# hadoop all in one image
+######################################################################
+
+ARG IMAGE_BASE=centos:7
+
+FROM ${IMAGE_BASE} as linkis-ldh
+
+ARG JDK_VERSION=1.8.0-openjdk
+ARG JDK_BUILD_REVISION=1.8.0.332.b09-1.el7_9
+ARG MYSQL_JDBC_VERSION=5.1.49
+
+ARG HADOOP_VERSION=2.7.0
```

Comment:
It is best to match the version numbers of hive, hadoop, and spark used by linkis by default